certified-connectors/PIPware KPIs/README.md

```diff
@@ -0,0 +1,30 @@
+
+
+## PIPware Connector
```

Comment:
@PIPwareInfo Please remove the empty strings in line number 1 and 2

Reply:
Hi @sriyen-msft  this is now resolved